### PR TITLE
Update index.html (added closing section tag)

### PR DESCRIPTION
**Pull request by Hector-Alexander-Lopez**
(added closing section tag)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,6 +20,7 @@ <h1>Our Time Tracker</h1>
 			<input type="text" name="task-input" class="task-input" placeholder="What are you working on?" />
 			<input type="submit" class="add-task-submit" value="Add Task" />
 		</form>
+			</section>
```
Comment (portfedh) [inline index.html:22]: This is an inline comment. I just discovered I can make some to your code

**Review by portfedh — CHANGES_REQUESTED** [review]
This is a general comment of the review of you code. 

I get three options:
- comment:Submit general feedback without explicit approval.
- Approve: Submit feedback and approve merging these changes.
-  Request changes : Submit feedback that must be addressed before merging.

Im choosing request changes. 
Please make any other change so we pretend that was a change request and ill approve it.

File: index.html (hunk 1 above)
Comment (Hector-Alexander-Lopez) [reply index.html:22]: Pretty cool feature. This is helpful

**Review by Hector-Alexander-Lopez — COMMENTED** [review]
Closed up an empty line between the body and h1

**Comment by Hector-Alexander-Lopez** [comment]
closed up an empty line between the body and h1

**Review by portfedh — APPROVED** [review]

**Pull request merged**